File: phaseGateOne/taskTwo.py
number =int(input("Enter a number between 0 and 1000:"))
firstInteger = int(number%10)
secondInteger  = int(number%100/10)
thirdInteger = int(number/100)
sum = firstInteger + secondInteger+thirdInteger
print(f"The sum of the numbers is:{sum}") 

File: phaseGateOne/TaskTwo.java
import java.util.Scanner;
public class TaskTwo{
public static void main(String[] args){
Scanner scanner = new Scanner(System.in);

System.out.print("Enter a number between 0 and 1000:");
int number = scanner.nextInt();

int firstInteger = number%10;
int secondInteger  = number%100/10;
int thirdInteger = number/100;

int sum = firstInteger + secondInteger+thirdInteger;

System.out.print("The sum of the number is "+ sum);

}
}